examples/tests/if.earth

```diff
@@ -0,0 +1,131 @@
+FROM alpine:3.13
```

Comment:
I think it would be good to also try this test with LOCALLY targets.

Reply:
True... I'll need to do that in a future PR.

Reply:
https://github.com/earthly/earthly/pull/794